src/staticanalysis/bot/static_analysis_bot/__init__.py

```diff
@@ -121,6 +121,20 @@ class Reliability(enum.Enum):
     Medium = 'medium'
     Low = 'low'
 
+    def invert(self):
```

Comment:
I would set this as a `@property`, named `inverted`, so that it's obvious to developers that it does not change any value on the class:

``` 
@property
def inverted(self):
  ...
  return ...
```

You can then use it as :`self.reliability.inverted` later on